components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -147,14 +148,20 @@ end
 --- Parses the team aliases, used in cases where a team is picked up by an org or changed
 --- name in some of the tournaments, in which case aliases are required to correctly query
 --- the team's results & points
-function AutomaticPointsTable:parseAliases(team, tournamentCount)
+function AutomaticPointsTable:parseAliases(team, tournamentCount, resolveRedirect)
 	local aliases = {}
+	local parseAlias = function(x)
+		if (resolveRedirect) then
+			return mw.ext.TeamLiquidIntegration.resolve_redirect(x)
+		end
+		return mw.language.getContentLanguage():ucfirst(x)

```

Comment:
```suggestion
		if (resolveRedirect) then
			return mw.ext.TeamLiquidIntegration.resolve_redirect(x)
		else
			return mw.language.getContentLanguage():ucfirst(x)
		end
```
RR already upercases first char so for that case an additional call on `ucfirst` is redundant

Reply:
ah yeah my bad didn't see the return
reviewing on phone is not good^^